test/generators/UserAnswersEntryGenerators.scala

```diff
@@ -260,7 +268,12 @@ trait UserAnswersEntryGenerators {
       case CountryOfDispatchPage             => arbitrary[Country].map(Json.toJson(_))
       case TransportedToSameCountryYesNoPage => arbitrary[Boolean].map(JsBoolean)
       case ItemsDestinationCountryPage       => arbitrary[Country].map(Json.toJson(_))
+      case ContainerIndicatorPage            => arbitrary[Boolean].map(JsBoolean)
     }
   }
 
+  private def generateTransportMeansAnswer: PartialFunction[Gettable[_], Gen[JsValue]] = {
+    case InlandModePage => arbitrary[InlandMode].map(Json.toJson(_))
```

Comment:
Not a huge issue but could be worth sticking to the pattern of popping the page import in here to keep to the existing pattern. Line 264 can also be changed to `import pages.transport.preRequisites._`